src/components/shared/card.tsx

```diff
@@ -150,10 +152,67 @@ const cardStyles = (itemType: RelatedItemType, format: Format): SerializedStyles
     }
 }
 
-const Card = ({ relatedItem, image }: Props): JSX.Element => {
+const parentIconStyles: SerializedStyles = css`
+    display:inline-block;
+    svg {
+        width: 0.875rem;
+        height: auto;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: 0.300rem;
```

Comment:
If we make the svg 1rem and the height of the parent 1.5rem, I think a margin-top of 0.25rem will centre it nicely.